src/twisted/web/test/test_web.py

```diff
@@ -957,6 +957,28 @@ def getContentFile(length):
         self.assertEqual([12345], lengths)
         self.assertIs(contentFile, request.content)
 
+    def test_parseBodySetFromSite(self):
+        """
+        C{Request._parseBody} is set to match C{Site._parseBody}.
```

Comment:
I feel compelled to note that this is a kind of crummy implementation-detail test but unless you immediately have an idea for improving it it's probably fine, I can see how you got here.